libut/tests/Makefile

```diff
@@ -4,7 +4,8 @@ OBJS=$(patsubst %,%.o,$(PROGS))
 
 CFLAGS += -I../include
 CFLAGS += -g
-CFLAGS += -Wall -Wextra
+CFLAGS += -Wall -Wextra -Werror -fPIC
+CFLAGS += $(OPTFLAGS)
```

Comment:
Yes, SUSE/openSUSE didn't install libut.a either. The build has it, but we deleted it later. I think it's common that Linux distributions don't ship static libraries. But things may be different under other cases.

About fPIC, I have little knowledge of it. But I think it's almost a common principle across distribution (position independent codes). I saw many downstream patches adding this in my packaging life. I found this explains it well: http://stackoverflow.com/questions/5311515/gcc-fpic-option